lightstep/tracer.py

```diff
@@ -47,18 +47,29 @@ def Tracer(**kwargs):
         inject/extract format (which relies on protobufs and may cause problems
         if other versions of protobufs are active in the same packaging
         configuration). Defaults to False (i.e., binary format is enabled).
+    :param ScopeManager scope_manager: the ScopeManager responsible for
+        Span activation. Defaults to the implementation provided by the
+        basictracer package, which uses thread-local storage.
     """
     enable_binary_format = True
     if 'disable_binary_format' in kwargs:
         enable_binary_format = not kwargs['disable_binary_format']
         del kwargs['disable_binary_format']
-    return _LightstepTracer(enable_binary_format, Recorder(**kwargs))
+
+    scope_manager = None
+    if kwargs.has_key('scope_manager'):
```

Comment:
`has_key` was removed in python 3.x https://docs.python.org/3.1/whatsnew/3.0.html#builtins

Please use `in`